compiler-rt/lib/rtsan/rtsan_interceptors_posix.cpp

```diff
@@ -244,6 +244,22 @@ INTERCEPTOR(int, close, int filedes) {
   return REAL(close)(filedes);
 }
 
+INTERCEPTOR(int, chdir, const char *path) {
+  __rtsan_notify_intercepted_call("chdir");
+  return REAL(chdir)(path);
+}
+
+INTERCEPTOR(int, fchdir, int fd) {
+  __rtsan_notify_intercepted_call("fchdir");
+  return REAL(fchdir)(fd);
+}
+
+INTERCEPTOR(int, chroot, const char *path) {
+  __rtsan_notify_intercepted_call("chroot");
+  return REAL(chroot)(path);
+}
```

Comment:
Could we separate out chroot? I think we should consider it separately considering it is a sudo only call. 

The other two interceptors look great, and I will happily approve those. Just want to consider chroot separately considering it's system-wide effects